web/client/epics/widgets.js

```diff
@@ -301,7 +312,37 @@ export const onWidgetCreationFromMap = (action$, store) =>
             const state = store.getState();
             const layer = getWidgetLayer(state);
             if (layer) {
-                observable$ = Rx.Observable.of(onEditorChange('chart-layers', [layer]));
+                observable$ = Rx.Observable.of(
+                    onEditorChange('chart-layers', [layer])
+                );
+            }
+            return observable$;
+        });
+
+
+export const onLayerSelectedEpic = (action$, store) =>
+    action$.ofType(EDITOR_CHANGE)
+        .filter(({key}) => key === 'chart-layers' && isDashboardEditing(store.getState()))
+        .switchMap(() => {
+            let observable$ = Rx.Observable.empty();
```

Comment:
This "default" value is never used.

Reply:
ok I will reformat, but note that this codestyle was copied from the precedent epic